packages/accordion/src/AccordionItem.ts

```diff
@@ -80,8 +91,10 @@ export class AccordionItem extends Focusable {
                 >
                     ${this.label}
                 </button>
-                <sp-icon-chevron100
-                    class="indicator spectrum-UIIcon-ChevronRight100"
+                <span class="iconContainer">
+                    <sp-icon-chevron100 class="indicator ${
```

Comment:
If you use `SizedMixin` as I indicated in my previous comments, you might be able to remove `chevronClass[this.size as DefaultElementSize]` and just have `indicator` be the only class. If that doesn't work, I would maybe try to also use the class `spectrum-UIIcon-ChevronRight100`.